meson_options.txt

```diff
@@ -0,0 +1,4 @@
+option('blas', type: 'string', value: '',
```

Comment:
According to https://mesonbuild.com/Build-options.html#build-options, this file should be renamed "meson.options" for meson version 1.1+. (Both seem to work, but the old name could get deprecated at some point). If we use meson.options, then we should need to ensure that the proper version of meson-python is being used, which is 0.14.0+